src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
Right, but what if the slave *is* configured to get an IPv6 address (RA/DHCP) -- as `link_ipv6_enabled()` always returns  False for bridge slaves this breaks this use case.

IMHO it shouldn't be `link_ipv6_enabled()`'s business to make such decisions, this should happen in the caller (like in your patch here). So this needs cleaning up.